sound/soc/sof/ipc4.c

```diff
@@ -599,6 +599,8 @@ static int ipc4_fw_ready(struct snd_sof_dev *sdev, struct sof_ipc4_msg *ipc4_msg
 	sdev->debug_box.offset = snd_sof_dsp_get_window_offset(sdev,
 							SOF_IPC4_DEBUG_WINDOW_IDX);
 
+	sdev->fw_ready.flags = ipc4_msg->extension;
```

Comment:
@RanderWang why do we need to introduce boot flags? Is this supported by the reference firmware? Can we use the FW config like we've done before?

Reply:
@ranj063 we don't have such module config query to support it now.  A new module config query will change IPC4 interface and affect windows driver.

Reply:
This PR is inspired by IPC3